ons_alpha/core/blocks/featured_document.py

```diff
@@ -77,6 +84,11 @@ class Meta:
     def get_context(self, value, parent_context=None):
         context = super().get_context(value, parent_context=parent_context)
         page = value["page"].specific
-        context["link_title"] = page.headline or page.title
+        context["link_title"] = page.headline or page.display_title
```

Comment:
What is `display_title`? Is that the admin-facing thing that uses the title of the most recent draft? If so, `page.title` is what we want here, as that's the official public-facing label.

Reply:
See the change that Mebin made this morning: https://github.com/ONSdigital/dis-wagtail-alpha/pull/134

Reply:
(this was after a long conversation about how the bulletin page title should appear that we had yesterday in stand-up)

Reply:
Okay, cool! Thanks for clarifying.